src/log.h

```diff
@@ -25,32 +25,39 @@
 
 namespace rhost {
     namespace log {
+        enum class log_verbosity {
```

Comment:
A different name sounds like a good idea then, for both of them - `log_level` should probably become `log_verbosity` to make it clear that it's just about how chatty we are, and `log_verbosity` should be `log_category`, and have specific enum levels for every category that we need to distinguish for security or PII purposes (i.e. no `minimal` or `full`, but more in the vein of `traffic` - e.g. `general_info` for things like version numbers or available memory, `user_input` for things like error messages that include expressions coming form the user, `user_data` for crash dumps etc).